src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -146,7 +144,7 @@ public string As
         /// <summary>
         /// This specifies a full or partial URI
         /// for the CSS information.
-        /// The html should reference the css file specified
+        /// The html should reference the css file specified.
```

Comment:
Suggest: Upper case HTML and CSS - consistency with the rest of the summary comment.

Reply:
Fixed.